dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/PacBioNotificationDto.java

```diff
@@ -9,6 +10,8 @@ public class PacBioNotificationDto extends NotificationDto {
 
   private Map<String, String> poolNames;
 
+  private List<Map<String, String>> positionList;
```

Comment:
Create a proper model class for this instead of using a map to write a set of undefined fields